geopyspark/tests/vector_pipe/feature_cell_value_schema_test.py

```diff
@@ -1,59 +0,0 @@
-import unittest
-import pytest
-from shapely.geometry import Point, LineString, MultiLineString
-
-from pyspark import RDD
-from pyspark.serializers import AutoBatchedSerializer
-from geopyspark.vector_pipe import Feature, CellValue
-from geopyspark.geotrellis.protobufserializer import ProtoBufSerializer
-from geopyspark.vector_pipe.vector_pipe_protobufcodecs import (feature_cellvalue_decoder,
-                                                               to_pb_feature_cellvalue,
-                                                               feature_cellvalue_encoder)
-from geopyspark.tests.base_test_class import BaseTestClass
-
-
-class FeatureCellValueSchemaTest(BaseTestClass):
```

Comment:
These look like they move to `geopyspark/geotrellis/protobufcodecs.py`. Can this test be carried forward?

Reply:
Yeah, it looks like we can still keep that test.